requirements.txt

```diff
@@ -1,5 +1,4 @@
-pytorch-lightning==1.2.0rc0  # todo: we shall align with real 1.2
-torch>=1.7  # TODO: regenerate weights with lewer PT version
+https://github.com/PyTorchLightning/pytorch-lightning/archive/flash_predict_step.zip
```

Comment:
?

Reply:
We were making changes on this branch to make it work. Just got merged into master.

Reply:
Switched to master.